src/locales/it.js

```diff
@@ -18,6 +18,281 @@ export default {
     'app.unpublish': 'Depubblica',
     'app.en': 'en',
     'app.it': 'it',
+    'app.aa': 'aa',
```

Comment:
All these codes don't need a translation. Why are you including them in the translation files? Wouldn't a constant be enough?